test/set_test.js

```diff
@@ -0,0 +1,153 @@
+goog.module('datalayerhelper.helper.testing.set');
```

Comment:
Can you call this file "setCommand_test" instead of just "set"? "set" by itself has multiple meanings that would be clarified with the new name.

It *might* also be worth updating "get_test.js" to be "getCommand_test.js", in a separate PR.